frontend/src/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import TranslateIcon from "@material-ui/icons/Translate";
+import "./DropdownMenu.css";
+
+function DropdownMenu() {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div className="DropdownMenu">
+      <Button
+        aria-controls="simple-menu"
+        aria-haspopup="true"
+        color="primary"
+        onClick={handleClick}
+      >
+        More
```

Comment:
What if we replace this text with a MenuIcon?

Reply:
Thanks for the comments,  I will fix them now!

Reply:
Hi Ted, I modified the code, not sure if that follow you suggestions, please correct me if I am wrong!

Reply:
I think your solution is generally very clean, readable, and functional across mobile and web  🙌  
Thanks for incorporating the feedback.